click/_winconsole.py

```diff
@@ -209,13 +233,21 @@ def _get_text_stdin(buffer_stream):
 
 
 def _get_text_stdout(buffer_stream):
+    if PY2:
+        buffer_stream = StreamWrapper(buffer_stream)
+        sys.stdout = StreamWrapper(sys.stdout)
```

Comment:
Is there a way to do this only for Windows versions that need it? Tested this on a Windows 10 VM and it wasn't necessary.

Reply:
[`sys.getwindowsversion`](https://docs.python.org/2.7/library/sys.html#sys.getwindowsversion) should do the trick. Assuming we don't need to detect Windows 8.1 (See [GetVersionEx](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724451(v=vs.85).aspx), because Microsoft, never understood why they did that). I guess the underlying limitation was fixed by Windows 8 condrv. Need to test if that's the right cutoff.

Reply:
I'm downloading an 8.1 VM (8 isn't available from MS), I can report on this later today.

Reply:
The issue does not occur in 8.1 or 10.

Reply:
So I'd bet it also doesn't happen in 8 and is likely fixed by condrv that was added in 8.